src/client/components/pages/about.js

```diff
@@ -110,7 +115,8 @@ function AboutPage() {
 			<p>
 				The core components of BookBrainz are the site, the
 				webservice and the database. All of these components are
-				running on MetaBrainz's Sakura server, which also runs&nbsp;
```

Comment:
It improved the readability of the code. @LordSputnik  has approved these changes you can see his 3rd comment from top for reference :)